src/zb_links/api/link/links.py

```diff
@@ -18,11 +18,11 @@
 
 search_by_arguments = reqparse.RequestParser()
 
-search_by_arguments.add_argument("author", type=str, required=False)
+search_by_arguments.add_argument("authors", type=str, required=False)
```

Comment:
Can you please split this pull request into two. One for the simple renaming and one for the rest. This simplifies reviewing significantly.

Reply:
If you insist on not splitting the PR, I am still interested in an argument why renaming of the fields was performed. Why are the new names better than the old ones. In general names with spaces are in some situations harder to use from the commandline etc.

Reply:
There was no defined strategy in changing the names. In some cases, like author-->authors I just followed the format used at https://zbmath.org/. If you believe that some choices are particularly bad, we can change them.

Reply:
has this been resolved?

Reply:
Can you make a short presentation of the final names at the next Monday meeting. If no-one objects to the proposed names I think it is good to go.

Reply:
Is that really appropriate for the Montagsrunde?  

Are the names even finalized?  We discussed with Fabian yesterday to add a zbl code as possible input, in which case a name would change.  Why bring it up now to everyone?

Reply:
Ok. So why changing the field names at all? It is a breaking change in the API. However, if there is a set of 'final' fieldnames this should be consistent among all APIs and user interfaces and IMHO worth discussing in the plenary session. Again I think this should be a separate issue.

Reply:
If there a list of finalized consistent fieldnames which have already been accepted, let's just use those.  But if we are defining those, let's first finalize them.  This would require knowledge of what other API fields are, consideration of empty spaces, whether we want to use zbl_code along with de numbers...

Reply:
I agree that the field names must be the same for all APIs we are deploying. At present, as also Fabian said yesterday, there is still no agreement about the standard names to use. When I changed the names in this branch, namely

author-->authors
classification-->MSC code
document-->DE number
external_id-->external id
type-->partner

I was only hoping to do my best to make things more readable for the current version, that is still in progress and subject to changes. I did not look at this change of names as something as a breaking change. This was not at all my intention. Sorry if you interpreted this as such. Probably the easiest way out is to discuss all together (Fabian included) to end up with some universal standard to use in all APIs. I'm totally open to other names and to any discussion about it.